sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/implementation/models/RoleDefinitionFilter.java

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.security.keyvault.administration.implementation.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/** The RoleDefinitionFilter model. */
+@Fluent
+public final class RoleDefinitionFilter {
```

Comment:
This should not be public. See my comments on the other "*Filter" class for details.

Reply:
This is auto-generated code and is in an internal package. It is also never used in any public surface APIs.